website_signup_legal_page_required/controllers/exceptions.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+
```

Comment:
add your own copyright for ordinary Python files as in https://github.com/OCA/maintainer-tools/blob/master/template/module/models/model_name.py